src/System.Management.Automation/engine/hostifaces/PowerShellProcessInstance.cs

```diff
@@ -11,6 +11,7 @@
 namespace System.Management.Automation.Runspaces
 {
     /// <summary>
+    /// This class represents a PowerShell process that is used for an out-of-process remote Runspace.
```

Comment:
There are still empty XML tags in this file.

Reply:
Updated the tags on public members.